classes/class3/HW3/catmodel/mongoosemodel.js

```diff
@@ -0,0 +1,11 @@
+var mongoose = require('mongoose');
```

Comment:
When we start having more than one model, you'll want to give each model file a more specific name. Unless your app is about mongooses instead of cats. :wink: